README.md

```diff
@@ -66,18 +66,17 @@ services:
         ports:
             - '3000:3000'
     mesh:
-        image: 0xorg/mesh:0xV3
+        image: 0xorg/mesh:7.2.1-beta-0xv3
         environment:
             ETHEREUM_RPC_URL: 'http://ganache:8545'
-            ETHEREUM_CHAIN_ID: '1337'
-            ETHEREUM_NETWORK_ID: '50'
```

Comment:
`ETHEREUM_NETWORK_ID` is not used by Mesh. See the [documentation about environment variables](https://0x-org.gitbook.io/mesh/getting-started/deployment#environment-variables).